gradio/components.py

```diff
@@ -1533,6 +1294,9 @@ def format_image(
                 + ". Please choose from: 'numpy', 'pil', 'filepath'."
             )
 
+    def generate_sample(self):
```

Comment:
Why do we need this method? As far as I can tell, it's only called outside components.py in test code?

Reply:
Ah yeah good point. This used to be called from the view API page in Gradio 2.x when we would generate a mock request for the user in the "Try It" section. It looks like we might bring it back (#1325) so I'd say let's leave this method here